deploy/scripts/src/txma/test.ts

```diff
@@ -35,9 +35,9 @@ const profiles: ProfileList = {
       startRate: 1,
       timeUnit: '1s',
       preAllocatedVUs: 1,
-      maxVUs: 350,
+      maxVUs: 300,
       stages: [
-        { target: 10, duration: '10m' } // Ramp up to 10 iterations per second in 10 minutes
+        { target: 30, duration: '10m' } // Ramp up to 10 iterations per second in 10 minutes
```

Comment:
```suggestion
        { target: 30, duration: '10m' } // Ramp up to 30 iterations per second in 10 minutes
```
Update comments to match values

Reply:
Done